MediaBrowser.MediaEncoding/Subtitles/OpenSubtitleDownloader.cs

```diff
@@ -63,24 +62,26 @@ void _config_NamedConfigurationUpdating(object sender, ConfigurationUpdateEventA
                 !string.IsNullOrWhiteSpace(options.OpenSubtitlesPasswordHash) &&
                 !options.OpenSubtitlesPasswordHash.StartsWith(PasswordHashPrefix, StringComparison.OrdinalIgnoreCase))
             {
-                options.OpenSubtitlesPasswordHash = EncryptPassword(options.OpenSubtitlesPasswordHash);
+                options.OpenSubtitlesPasswordHash = EncodePassword(options.OpenSubtitlesPasswordHash);
             }
         }
 
-        private string EncryptPassword(string password)
+        private static string EncodePassword(string password)
         {
-            return PasswordHashPrefix + _encryption.EncryptString(password);
+            var bytes = Encoding.UTF8.GetBytes(password);
+            return PasswordHashPrefix + bytes;
```

Comment:
Won't that cause db incompatibility?

Reply:
Copy/paste fail. The OS password is just saved as a base64 encoded string in config/subtitles.xml, but seems I forgot the encoding part.